specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/Recommendations.json

```diff
@@ -463,77 +360,37 @@
             "description": "Recommendation is active."
           },
           {
-            "value": "Disabled",
-            "description": "Recommendation is disabled."
-          },
-          {
-            "value": "CompletedByUser",
-            "description": "Recommendation has been completed by user."
+            "value": "InProgress",
+            "description": "Recommendation is in progress."
           },
           {
-            "value": "CompletedByAction",
-            "description": "Recommendation has been completed by action."
+            "value": "Dismissed",
+            "description": "Recommendation has been dismissed."
           },
           {
-            "value": "Hidden",
-            "description": "Recommendation is hidden."
-          }
-        ]
-      }
-    },
-    "RecommendationPriority": {
-      "description": "Priority of recommendation.",
-      "type": "string",
-      "enum": [
-        "Low",
-        "Medium",
-        "High"
-      ],
-      "x-ms-enum": {
-        "name": "Priority",
-        "modelAsString": true,
-        "values": [
-          {
-            "value": "Low",
-            "description": "Low priority for recommendation."
-          },
-          {
-            "value": "Medium",
-            "description": "Medium priority for recommendation."
+            "value": "CompletedByUser",
+            "description": "Recommendation has been completed by user."
           },
           {
-            "value": "High",
-            "description": "High priority for recommendation."
+            "value": "CompletedBySystem",
+            "description": "Recommendation has been completed by the system."
           }
         ]
       }
     },
-    "ActionState": {
-      "description": "Represents the state the recommendation action is currently in.",
-      "type": "string",
-      "enum": [
-        "Active",
-        "InProgress",
-        "Done"
+    "ReevaluateResponse": {
+      "description": "Reevaluate response object.",
+      "properties": {
+        "lastEvaluatedTimeUtc": {
+          "description": "The time stamp (UTC) when the recommendation was last evaluated.",
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": [
+        "lastEvaluatedTimeUtc"
       ],
-      "x-ms-enum": {
-        "name": "Priority",
-        "modelAsString": true,
-        "values": [
-          {
-            "value": "Active",
-            "description": "Default state of actions."
-          },
-          {
-            "value": "InProgress",
-            "description": "State for when actions have been clicked on."
-          },
-          {
-            "value": "Done",
-            "description": "State of completed actions."
-          }
-        ]
-      }
+      "type": "object"
     }
   },
   "parameters": {
```

Comment:
Just noticed in the example file that the patch request is an array
![image](https://github.com/Azure/azure-rest-api-specs/assets/49263932/f9b72a37-a057-4070-aa4e-bd04fa224b5d)
![image](https://github.com/Azure/azure-rest-api-specs/assets/49263932/016e6e12-aed9-4662-964d-7e76eda76a9d)

it should be an object, right?
![image](https://github.com/Azure/azure-rest-api-specs/assets/49263932/3d553c7a-74d9-4c5f-a483-f3cebb451445)

Reply:
We actually have it twice, I'm not sure why they did it this way initially:
![image](https://github.com/Azure/azure-rest-api-specs/assets/19761142/e2754e46-f052-49d5-834f-ac3e675b3c5b)

Reply:
Changed, LMK what you think.